azurerm/internal/services/web/tests/app_service_environment_resource_test.go

```diff
@@ -379,3 +379,39 @@ resource "azurerm_subnet" "gateway" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger)
 }
+
+func TestAccAzureRMAppServiceEnvironment_internalLoadBalancer(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_app_service_environment", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMAppServiceEnvironmentDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMAppServiceEnvironment_internalLoadBalancerAndWhitelistedIpRanges(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMAppServiceEnvironmentExists(data.ResourceName),
+					resource.TestCheckResourceAttr(data.ResourceName, "internal_load_balancing_mode", "Web, Publishing"),
+				),
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
```

Comment:
Can we have this at the top to match the layout in the other test definitions? The flow usually runs: Test definitions, Test Functions, Test Configs.

Reply:
Thanks! I updated the documentation and code. -> Removal of possible deprecation note, moving test definition to match code layout